csi-plug-n-play/sidecars/soda-csi-provisioner/external-provisioner/cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -161,7 +161,7 @@ func main() {
 	if err != nil {
 		klog.Fatalf("Error getting CSI driver name: %s", err)
 	}*/
-	provisionerName := "soda-csi-block"
+	provisionerName := "soda-csi"
```

Comment:
So this can work for file and block?

Reply:
yes, the file and block specs will be coming from profile